RadixWallet/Clients/AccountPortfoliosClient/AccountPortfoliosClient+Live.swift

```diff
@@ -25,12 +25,21 @@ extension AccountPortfoliosClient: DependencyKey {
 			}
 		}
 
+		/// Update when hidden assets change
+		Task {
+			for try await _ in await appPreferencesClient.appPreferenceUpdates().map(\.assets.hiddenAssets) {
```

Comment:
Doesn't this actually get called every time _anything_ changes in appPreferences? Not a big deal probably, but you might as well throw on a `removeDuplicates()`.

Reply:
great catch!